components/UnitFiles.vue

```diff
@@ -124,12 +121,6 @@ export default {
     }
   },
   watch: {
-    $props: {
-      immediate: true,
-      handler() {
-        this.validateProps()
```

Comment:
I removed it for 2 reasons:
-  I found it a bad practice to validate values coming directly from the manifest only once the `UnitFiles` component is loaded. It's better to do these checks in the `manifests/index.js` file so that we detect errors without having to load on each individual experience.
- Most of the checks in `validateProps` didn't make sense for the new requirements that we have: for the Google My Activity experience, we need to be able to extract all the files from a zip file and load some of them based on a regex pattern, but we could not do this using the `file` property, and it is done in a custom pipeline instead.

Although I recognize that we should write new checks based on the changes that we made.